### ISSUE TITLE
Permission Denied on user-media/addons directories on fresh local env

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#9659]
- Call `make initialize_docker` locally

This will do a bunch of stuff, but most importantly `initialize_db` will wipe `user-media/` directory and `populate_data` will create new add-ons, creating some files & directories under `user-media/` in the process.

Now, because we're doing this from the container shell, it will be done with the root user. The permissions will end up looking like this on my system:
```
/bin/ls -l user-media/addons 
total 72
drwxr-xr-x 2 root root 4096 oct.  11 12:32 21
(...)
drwxr-xr-x 2 root root 4096 oct.  11 12:33 86
drwxr-xr-x 2 root root 4096 oct.  11 12:33 temp
```

This is bad, because the `runserver` and `supervisord` on the `web` and `worker` containers respectively use the `olympia` user, not `root`. So they can't move files to those directories, unless somehow the `olympia` user is the same uid as the `root` one (which may happen depending on your docker configuration ? I'm not sure, but it's not the case in mine)

**Comment by diox** [comment]
Most commands in our Makefiles should be run as `olympia`, so the fix would be to make them do that... but let's be careful because some might actually require `root` access on the container (for things like installing deps... although we could fix that as well, maybe ?)

**Comment by EnTeQuAk** [comment]
I'm looking at this as part of #12366

**Comment by EnTeQuAk** [comment]
Fixed as part of #12366

**Comment by AlexandraMoga** [comment]
Please add a qa:not needed label to the issue.
Thanks
